### ISSUE TITLE
Breaking change in @ethersproject/sha2-beta.134 causes error when authenticating

### ISSUE THREAD

**Issue by msterle** [3box/identity-wallet-js#30]
When instantiating an IdentityWallet with a seed, the following error occurs

```
UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'sha512' of undefined
```

Sample code:
```
const IdentityWallet = require('identity-wallet')
const seed = '0x95838ece1ac93a51b68823b21ce9f564bc536eebb9c3500fa6da81f17086a6be'
const idWallet = new IdentityWallet(() => true, { seed })
idWallet.authenticate()
```

This seems to occur in the `@ethersproject/sha2` package - in version beta.134, the top-level member `SupportedAlgorithms` was renamed `SupportedAlgorithm`

**Comment by msterle** [comment]
While this is being resolved, a workaround that may work is to explicitly include the correct version of the @ethersproject/sha2 package in projects that use this by running `npm install @ethersproject/sha2@5.0.0-beta.133`

**Comment by ebadiere** [comment]
> While this is being resolved, a workaround that may work is to explicitly include the correct version of the @ethersproject/sha2 package in projects that use this by running `npm install @ethersproject/sha2@5.0.0-beta.133`

This fix worked for me.  Thanks!

**Comment by oed** [comment]
This should be fixed by updating `@ethersproject/hdnode` and `@ethersproject/wallet` to the latest version.

**Comment by msterle** [comment]
I tried that, but it seems that the build of beta.134 is inconsistent - hdnode and sha2 are incompatible for that version.

**Comment by oed** [comment]
What I have installed that works:
```
    "@ethersproject/hdnode": "5.0.0-beta.137",
    "@ethersproject/wallet": "5.0.0-beta.136",
```

**Comment by msterle** [comment]
fixed by #32
